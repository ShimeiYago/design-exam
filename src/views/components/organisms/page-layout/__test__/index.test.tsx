import { shallow, ShallowWrapper } from 'enzyme';
import { PageLayout, Props } from '..';

let wrapper: ShallowWrapper<Props>;

const props: Props = {
  children: 'children',
  sidebar: 'sidebar',
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<PageLayout {...props} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
