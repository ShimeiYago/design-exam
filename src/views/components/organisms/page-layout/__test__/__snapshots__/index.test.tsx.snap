// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<div
  className="container"
>
  <div
    className="sidebar"
  >
    <Card
      minHeight="calc(100vh - 48px)"
    >
      sidebar
    </Card>
  </div>
  <div
    className="main"
  >
    <Card
      radius={true}
    >
      children
    </Card>
  </div>
</div>
`;
